cmd/kubeadm/app/util/staticpod/utils.go

```diff
@@ -220,6 +220,15 @@ func ReadStaticPodFromDisk(manifestPath string) (*v1.Pod, error) {
 
 // LivenessProbe creates a Probe object with a HTTPGet handler
 func LivenessProbe(host, path string, port int, scheme v1.URIScheme) *v1.Probe {
+	return createHTTPProbe(host, path, port, scheme, 15, 15, 8)
+}
+
+// ReadinessProbe creates a Probe object with a HTTPGet handler
+func ReadinessProbe(host, path string, port int, scheme v1.URIScheme) *v1.Probe {
+	return createHTTPProbe(host, path, port, scheme, 15, 15, 8)
```

Comment:
I left here the defaults of the `livenessProbe` we could also change these values to check the readiness more frequently.